src/config/dynamic-theme-fixes.config

```
@@ -12125,20 +12125,30 @@ INVERT
 .add-more-forum
 .all-wraper
 .core_title_btns
+.day_rcmd > .class_title
+.fengchao-wrap-box span
 .focus_btn
+img[src*="sign_err.png"]
 .nav_wrap_add_border
 .save_face_bg
 .share_btn_wrapper
+.sub_nav_wrap
+.threadlist_rep_num
+.u-f-item
 
 CSS
+.card_banner {
+    opacity: 50%;
+}
 .class_title,
 .content-sec,
-.f-d-w,
+.f-d-item,
 .left-sec,
+.page-container,
+.pb_footer,
 .sub_nav_wrap,
-.tb_footer,
-.u-f-item {
-    background-color: var(--darkreader-neutral-background) !important;
+.tb_footer {
+    background: none !important;
```

```suggestion
    background-image: none !important;
```
Specify it a bit further.